core/contracts/financial-templates/implementation/ExpiringMultiParty.sol

```diff
@@ -5,5 +5,5 @@ import "./Liquidatable.sol";
 
 
 contract ExpiringMultiParty is Liquidatable {
-    constructor(ConstructorParams memory params) public Liquidatable(params) {}
+    constructor(ConstructorParams memory params) public Liquidatable(params) nonReentrant() {}
```

Comment:
Does `nonReentrant()` go into effect before or after the Liquidatable constructor gets called? Is it possible to put this modifier _before_ the Liquidatable constructor is called? Or if it's not, maybe it makes sense to add `nonReentrant()` to all the constructors (since they shouldn't interfere with one another)?

Reply:
The constructors do not appear to interfere with each other so I just went ahead and put it on all of them.